public/crclib.c

```diff
@@ -112,65 +112,42 @@ void CRC32_ProcessByte( dword *pulCRC, byte ch )
 void CRC32_ProcessBuffer( dword *pulCRC, const void *pBuffer, int nBuffer )
 {
 	dword	ulCrc = *pulCRC, tmp;
-	byte	*pb = (byte *)pBuffer;
-	uint	nFront;
-	int	nMain;
-JustAfew:
-	switch( nBuffer )
+	dword	*pb = (dword *)pBuffer;
```

Comment:
What about alignment? Is it safe?

Reply:
why should there be alignment?

Reply:
https://developer.apple.com/documentation/code_diagnostics/undefined_behavior_sanitizer/misaligned_pointer